flink-formats/flink-parquet/pom.xml

```diff
@@ -73,6 +73,11 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
Seems like guava is not mandatory in your implementation.

Reply:
Used in unit test. Yeah, but is not mandatory.